taxcalc/functions.py

```diff
@@ -120,23 +179,82 @@ def ItemDed(_posagi, e17500, e18400, e18425, e18450, e18500, e18800, e18900,
                  ID_Charity_frt, puf):
 
     """
-    WARNING: Any additional keyword args, such as 'puf=True' here, must be 
+    Itemized Deduction; Form 1040, Schedule A
+    
+    Notes
+    -----
+    Tax Law Parameters:
+        ID_ps : Itemized deduction phaseout AGI start (Pease)
+        
+        ID_crt : Itemized deduction maximum phaseout as a percent of total itemized deduction (Pease)
+        
+        ID_prt : Itemized deduction phaseout rate (Pease)
+        
+        ID_Medical_frt : Deduction for medical expenses; floor as a percent of AGI
+        
+        ID_Casualty_frt : Deduction for casualty loss; floor as a percent of AGI
+        
+        ID_Miscellaneous_frt : Deduction for miscellaneous expenses; floor as a percent of AGI
+        
+        ID_Charity_crt_Cash : Deduction for charitable cash contributions; ceiling as a percent of AGI
+        
+        ID_Charity_crt_Asset : Deduction for charitable asset contributions; ceiling as a percent of AGI
+        
+        ID_Charity_frt : Deduction for charitable contributions; floor as a percent of AGI
+    
+    Taxpayer Charateristics:
```

Comment:
Spell check on characteristics (and intermediate variables, below), and we should be consistent w capitalization (Taxpayer Characteristics vs. Taxpayer characteristics).